cairo/src/helper.rs

```diff
@@ -1,34 +1,31 @@
 //! This module inlcudes some field helpers that are useful for Cairo
 
 use ark_ff::Field;
-use o1_utils::{field_helpers::i128_to_field, FieldHelpers};
+use o1_utils::FieldHelpers;
 
 //(TODO move to utils inside FieldHelpers)
 
 /// Field element helpers for Cairo
 pub trait CairoFieldHelpers<F> {
     /// Return field element as byte, if it fits. Otherwise returns least significant byte
-    fn lsb(self) -> u8;
+    fn of_lsb(self) -> u8;
 
-    /// Return pos-th 16-bit chunk as another field element
-    fn chunk_u16(self, pos: usize) -> F;
+    /// Return `pos`-th 16-bit chunk as another field element
+    fn of_u16_chunk(self, pos: usize) -> F;
```

Comment:
please let's avoid `of_` which is an OCaml thing :D

Reply:
Ah ok! Both sounded good to me, I didn't know there was a standard

Reply:
yeah I think I replied on the comment there. But in rust we use `from` and `into` and `to` generally. (Usually `into` is used when `self` is consumed and does not implement `Copy`)

Reply:
oh I was not aware of the `into` distinction for consuming the `self`! nice lesson